manifests/s/ShiningLight/OpenSSL/Light/3.4.1/ShiningLight.OpenSSL.Light.installer.yaml

```
@@ -11,38 +11,38 @@ InstallationMetadata:
   DefaultInstallLocation: '%Temp%\~_tmpdir'
 Installers:
 - Architecture: x86
-  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-3_4_0.msi
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL_Light-3_4_1.msi
   InstallerSha256: 548A9842A8212AA37655D84B24A4F6A3A2E56F72F28FF1676D0AFED5A33410EB
   Dependencies:
     PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x86
   ProductCode: '{6367BE5C-A43A-47BB-93E6-F7BA1C4078FB}'
   AppsAndFeaturesEntries:
-  - DisplayName: OpenSSL Light 3.4.0 (32-bit)
+  - DisplayName: OpenSSL Light 3.4.1 (32-bit)
     Publisher: Shining Light Productions
     ProductCode: '{6367BE5C-A43A-47BB-93E6-F7BA1C4078FB}'
     UpgradeCode: '{31D2408A-9CAE-4988-9EC3-F40FDE7D6AE5}'
 - Architecture: x64
-  InstallerUrl: https://slproweb.com/download/Win64OpenSSL_Light-3_4_0.msi
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL_Light-3_4_1.msi
   InstallerSha256: 517D9490014D3A9BD297693F32ADF4E96F2FC126FD657E05C54EF29013E0BF24
   Dependencies:
     PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
   ProductCode: '{C77635D0-5944-407D-A8D7-AC72DAF97AB1}'
   AppsAndFeaturesEntries:
-  - DisplayName: OpenSSL Light 3.4.0 (64-bit)
+  - DisplayName: OpenSSL Light 3.4.1 (64-bit)
     Publisher: Shining Light Productions
     ProductCode: '{C77635D0-5944-407D-A8D7-AC72DAF97AB1}'
     UpgradeCode: '{50A9FBE2-0F8C-4D5D-97A4-A63A71C4EA1E}'
 - Architecture: arm64
-  InstallerUrl: https://slproweb.com/download/Win64ARMOpenSSL_Light-3_4_0.msi
+  InstallerUrl: https://slproweb.com/download/Win64ARMOpenSSL_Light-3_4_1.msi
   InstallerSha256: 7B21D5C7630D75D67620C38C2192773CA488A7E9CF3AD6081855F49A006D8DF2

```

```suggestion
  InstallerSha256: 9A46A0ABB55F2116D8A6C7673978F2575D5203663C5F27E906E16AC9089CC04D
```

(Automated response - build 990.)